MonitoringSystem/Client/Types/BaseType.py

```diff
@@ -0,0 +1,149 @@
+########################################################################
+# $Id: $
+########################################################################
+
+"""
+Helper class for configuring the monitoring service. 
+
+"""
+
+__RCSID__ = "$Id$"
+
+########################################################################
+class BaseType( object ):
```

Comment:
maybe a less generic name than "BaseType" would be appropriate ? It is like avoiding to call yet another class File :-)